.github/workflows/auto_integrate.yml

```diff
@@ -35,6 +33,7 @@ jobs:
         uses: softprops/turnstyle@v1
```

Comment:
pin to specific commit ref instead of release version?

Reply:
No :-P Discussed offline, but the current security policy is just that we have to use a tagged release: https://opensource.google/docs/github/services/#actions

If the policy changes, we'll switch to a specific commit sha.